src/frontend/src/components/Post/Post.jsx

```diff
@@ -69,15 +70,14 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function formatPublishedDate(dateString) {
+const formatPublishedDate = (dateString) => {
```

Comment:
Why this change?

Reply:
Upgrading to ES6 all the codes I'm working on. Is this ok?

Reply:
In general, I would avoid a change like this if you're not updating that code, and do it in a dedicated "switch to ES6" type PR (i.e., isolate the changes from each other so the git history stays in tact with the change).  But it's not a big deal, just FYI.